src/main/java/Duke.java

```diff
@@ -1,10 +1,205 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.File;
+import java.io.FileNotFoundException;
+
+import messages.ErrorMessages;
+import messages.OperationsMessages;
+import tasks.Deadline;
+import tasks.Event;
+import tasks.Todo;
+import exceptions.EmptyInputException;
+import exceptions.InvalidCommand;
+
 public class Duke {
+    // Constants
+    private static final String BOT_NAME = "Duke"; 
+    private static final String saveFilePath = "./data/duke.txt";
+    private static ArrayList<Todo> itemList = new ArrayList<Todo>(0);
+    // private static Set<String> markedItems = new HashSet<String>(MAX_ITEMS);
```

Comment:
Code Quality: Avoid unsafe shortcuts,

Consider removing dead code. This might not have been covered yet in the module, but still good to take note of.